test/framework/src/main/java/org/elasticsearch/common/lucene/store/ESIndexInputTestCase.java

```diff
@@ -180,6 +181,15 @@ public void onRejection(Exception e) {
         return output;
     }
 
+
+    private static final AtomicLong uniqueIdGenerator = new AtomicLong();
```

Comment:
let's make this non-static, and initialize it on a per-test basis (i.e. add `@Before` method) so that it does not break reproducibility.

Reply:
++ done in 4322be8